### PR TITLE
Add refresh button to abtesting [SwiftUI]

### PR DESCRIPTION
**Pull request by jrcrespoh**
Add "Refresh" button to SwiftUI Quickstart for A/B Testing. Button queries RemoteConfig and updates UI accordingly. Static UI tests included.

UI with light color scheme
UI with dark color scheme

### CODE REVIEW

**Review by morganchen12 — APPROVED** [review]

**Review by paulb777 — APPROVED** [review]
LGTM!

**Review by jrcrespoh — COMMENTED** [review]
@morganchen12 @paulb777 requesting feedback/suggestions on wording of button purpose/instructions on README.md

File: abtesting/README.md [hunk 1]
```diff
@@ -18,7 +18,9 @@ Verify the following two flows:
 Test on device: 
 Run the sample and copy the printed installation auth token from Xcode's console
 into the 'Manage test devices' section in Firebase Console (click into details
-in 'Experiment Overview' when experiment is in Draft status).
+in 'Experiment Overview' when experiment is in Draft status). Verify you can
```
Comment (paulb777) [inline abtesting/README.md:21]: How about "Toggle the A/B test state by tapping the Refresh button."

Comment (jrcrespoh) [reply abtesting/README.md:21]: Hmmm that's what I was worried about... The button only reads from RemoteConfig and applies changes to the UI, if any. It does not actually switch you out of the A/B test. So repeated presses will log to console
```
Config fetch completed with status: success
Remote Config did not change from: dark
```
for example. How could I make that clearer?

Comment (paulb777) [reply abtesting/README.md:21]: I see...  How about something like:

"Tap the Refresh button to update the UI after making changes to the A/B test configuration on the console."

**Pull request merged**